packages/webview_flutter/webview_flutter/README.md

```diff
@@ -17,58 +17,78 @@ widget's Dartdoc for more details on how to use the widget.
-backend is used, this implementation has multiple
-[keyboard issues](https://github.com/flutter/flutter/issues?q=is%3Aopen+label%3Avd-only+label%3A%22p%3A+webview-keyboard%22).
-When keyboard input is required we recommend using the Hybrid Composition based platform views
-implementation. Note that on Android versions prior to Android 10 Hybrid Composition has some
-[performance drawbacks](https://flutter.dev/docs/development/platform-integration/platform-views#performance).
+the Android’s webview within the Flutter app. It supports two modes: *Virtual displays* (the current default) and *Hybrid composition*.
 
-### Using Hybrid Composition
+Here are some points to consider when choosing between the two:
+
+* *Hybrid composition* mode has a built-in keyboard support while *Virtual displays* mode has multiple
+[keyboard issues](https://github.com/flutter/flutter/issues?q=is%3Aopen+label%3Avd-only+label%3A%22p%3A+webview-keyboard%22)
+* *Hybrid composition* mode only requires Android SKD 19+ while *Virtual displays* mode requires Android SDK 20+
+* *Hybrid composition* mode has [perfomence limitations](https://flutter.dev/docs/development/platform-integration/platform-views#performance) when working on Android versions perior to Android 10 while *Virtual displays* is performent on all supported Android versions 
+
+|                             | Hybrid composition  | Virtual displays |
+| --------------------------- | ------------------- | ---------------- |
+| **Full keyboard supoport**  | yes                 | no               |
+| **Android SDK support**     | 19+                 | 20+              |
+| **Full performance**        | Android 10+         | always           |
+| **The default**             | no                  | yes              |
+
+### Using Virtual displays
 
-1. Set the `minSdkVersion` in `android/app/build.gradle`:
+The mode is currently enabled by default. You should however make sure to set the correct `minSdkVersion` in `android/app/build.gradle` (if it was previously lower than 20):
 
 ```groovy
 android {
     defaultConfig {
-        minSdkVersion 19
+        minSdkVersion 20
     }
 }
 ```
 
-This means that app will only be available for users that run Android SDK 19 or higher.
 
-2. To enable hybrid composition, set `WebView.platform = SurfaceAndroidWebView();` in `initState()`.
-For example:
-
-```dart
-import 'dart:io';
+### Using Hybrid Composition
 
-import 'package:webview_flutter/webview_flutter.dart';
+1. Set the correct `minSdkVersion` in `android/app/build.gradle` (if it was previously lower than 19):
 
-class WebViewExample extends StatefulWidget {
-  @override
-  WebViewExampleState createState() => WebViewExampleState();
-}
-
-class WebViewExampleState extends State<WebViewExample> {
-  @override
-  void initState() {
-    super.initState();
-    // Enable hybrid composition.
+    ```groovy
+    android {
+        defaultConfig {
+            minSdkVersion 19
+        }
+    }
+    ```
```

Comment:
Thanks, but they are not redundant - this is inside an enumerated list and the indentation is needed so that the code block remains the part of the correct list item.

Reply:
That makes sense, but then maybe also make sure the indentation is also used in the "Using Virtual displays" section?

Reply:
There is no list in the "Using Virtual displays" section (since it would be a one-element list)

Reply:
I don't understand, isn't it the exact same `android/app/build.gradle` file?

Reply:
> I don't understand, isn't it the exact same `android/app/build.gradle` file?

Nevermind, I understand now. It is part of the list in the `README.md`, sorry for the confusion.